src/components/views/Instance/Edit/Deployment.tsx

```diff
@@ -82,6 +83,30 @@ export function Deployment(): JSX.Element {
             setCompileJobsTotalPages(response.payload.totalPages);
             setCompileJobsPage(page);
             setCompileJobs(response.payload.content);
+
+            // also grab the job on the next page if possible
+            if (
+                response.payload.content.length == compileJobsPageSize &&
+                response.payload.totalPages > page
+            ) {
+                const extraResponse = await DreamMakerClient.listCompileJobs(
+                    instanceEditContext.instance.id,
+                    {
+                        page: page + 1,
+                        pageSize: compileJobsPageSize
```

Comment:
```suggestion
                        page: (compileJobsPageSize * page) + 1,
                        pageSize: 1
```

Avoiding overfetching

Reply:
Prettier didn't like the parens but otherwise fine.